lib/galaxy/authnz/custos_authnz.py

```diff
@@ -475,3 +460,118 @@ def _username_from_userinfo(self, trans, userinfo):
             return f"{username}{count}"
         else:
             return username
+
+
+class OIDCAuthnzBaseKeycloak(OIDCAuthnzBase):
+    def __init__(self, provider, oidc_config, oidc_backend_config, idphint=None):
+        super().__init__(provider, oidc_config, oidc_backend_config, idphint)
+        self.config.extra_params = {
+            "kc_idp_hint": oidc_backend_config.get("idphint", "oidc")
+        }
+        self._load_config()
+
+    def _get_provider_specific_scopes(self):
+        return ["org.cilogon.userinfo"]
```

Comment:
```suggestion
        return []
```

Reply:
Removed the function, since the default implementation returns [].